src/go/docker-credential-none/cmd/root.go

```diff
@@ -50,11 +50,20 @@ using the same format that docker uses when no credsStore field is specified in
 This helper is intended for testing purposes, but will be used on Linux systems
 unless 'pass' and/or 'secretservice' is available.`,
 		configFileName),
+	DisableSuggestions: true,
+	SilenceUsage:       true,
 }
 
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
+	//Do our own validation to
```

Comment:
This sentence appears to be incomplete. Presumably this will explain _why_ we can't use Cobra's default validation? (Also, space after `//` please.)